src/main/java/hvac/meetingontology/RequestStatus.java

```diff
@@ -0,0 +1,12 @@
+package hvac.meetingontology;
+
+public enum RequestStatus {
+    CANCELLED,
+    ESTIMATION,
+    EXECUTE,
+    FAILED,
+    FINISHED,
+    FORECAST,
+    OFFER,
+    RESERVED,
+}
```

Comment:
Może lepiej zmienić nazwę tego enuma na RequestType, RequestStatus bardziej kojarzy mi się z czymś co  jest odpowiedzią na Request. Chyba że to jest tymczasowo i potem nad lepszą ontologią popracujemy.

Reply:
Ogólnie problem jest taki, że nie chciałem zbytecznie komplikować ontologii według zasady KISS, więc ten enum powinien mówic, na jakim etapie/jaki status ma przesyłany meeting. Może faktycznie powinienem zmienić nazwę na Meeting Status?